docs/framework/cloud-deploy.mdx

```diff
@@ -1,167 +1,173 @@
+    URL: https://your_app_url
+    ```
+  </Accordion>
+
+  <Accordion title="writer cloud undeploy">
+    Stops the deployed app and makes it unavailable.
+
+    **Usage:**
+    ```
+    writer cloud undeploy [OPTIONS]
+    ```
+
+    **Options:**
+    - `--api-key`: Writer API key for authentication. If not provided, you will be prompted to enter it.
+    - `--verbose, -v`: Enable verbose output.
+
+    **Example:**
+    ```
+    writer cloud undeploy
+    ```
+
+    **Description:**
+    - Stops and removes the deployed app from the Writer cloud.
+  </Accordion>
+
+  <Accordion title="writer cloud logs">
+    Fetches logs from the deployed app.
+
+    **Usage:**
+    ```
+    writer cloud logs [OPTIONS]
+    ```
+
+    **Options:**
+    - `--api-key`: Writer API key for authentication. If not provided, you will be prompted to enter it.
+    - `--verbose, -v`: Enable verbose output.
+
+    **Example:**
+    ```
+    writer cloud logs
+    ```
+
+    **Description:**
+    - Continuously fetches and displays logs from the deployed app.
+    - Logs are ordered by date and time.
+
+    **Output Example**
+
+    ```
+    2024-06-11 09:27:02.190646+00:00 [INFO] Starting container entrypoint...
+    2024-06-11 09:27:03.798148+00:00 [INFO] BuildService - Downloading build files...
+    ...
+    ```
+  </Accordion>
+</AccordionGroup>
+
+## Environment variables
 
 When deploying an app, you can specify environment variables that will be available to the app during runtime. Use the `--env` option to pass these variables.
 
-**Example:**
 ```
 writer cloud deploy hello --env DB_HOST=db.example.com --env DB_PORT=5432
 ```
 
 In this example, `DB_HOST` and `DB_PORT` will be available to the app as environment variables.
 
-## API Key
+## API key
 
 The `WRITER_API_KEY` is crucial for deploying and managing apps. It is used for authenticating requests to the Writer cloud. If not provided as an option, the CLI will prompt you to enter it. The `WRITER_API_KEY` will also be added to the deployed app's environment to enable AI features of the Writer framework.
 
-## Deployment Process
-
-1. **Package Creation:** The CLI packages the app, excluding certain files (e.g., Dockerfile, `service_entrypoint.py`).
-2. **Upload Package:** The package is uploaded to the Writer deployment server.
-3. **Build and Deploy:** The server builds and deploys the app, making it accessible via a URL.
-4. **Environment Variables:** Specified environment variables are set, and `WRITER_API_KEY` is added by default.
-
-## Example Workflow
-
-1. **Deploying an App:**
-   ```
-   writer cloud deploy /path/to/app --env DB_HOST=db.example.com --env DB_PORT=5432
-   ```
-
-2. **Fetching Logs:**
-   ```
-   writer cloud logs
-   ```
-
-3. **Undeploying an App:**
-   ```
-   writer cloud undeploy
-   ```
+## Deployment process
+
+<Steps>
+  <Step title="Package Creation">
+    The CLI packages the app, excluding certain files (e.g., Dockerfile, `service_entrypoint.py`).
```

Comment:
@raaymax what's the "service_entrypoint.py"? That's ours, right? Should we disclose it?

Reply:
Also, I think this should be a comprehensive list, not two examples, since it's not obvious what the other excluded files are (if any)